src/mslice/models/workspacemanager/workspace_algorithms.py

```diff
@@ -25,6 +25,7 @@
 from mslice.workspace.pixel_workspace import PixelWorkspace
 from mslice.workspace.histogram_workspace import HistogramWorkspace
 from mslice.workspace.workspace import Workspace
+from mslice.workspace.helperfunctions import WorkspaceNameAppender
```

Comment:
Could we rename this `WorkspaceNameModifier` or something, as we will have to prepend, not just append?